lib/libutils/ext/include/asm.S

```diff
@@ -10,7 +10,19 @@
 #define UNWIND(...)
 #endif
 
-	.macro FUNC name colon section=default align=4
+/*
```

Comment:
So far we've managed to keep this file more or less architecture neutral, but this is a big step away from that. I'd rather move the non-generic parts into some other file, perhaps `asm_a64.S`

Reply:
Are you suggesting to move the BTI macro definition and macro emit_aarch64_feature_1_and to asm_a64.S ?

Reply:
Yes, please.